### PR TITLE
fits2isis tests now check the labels. Also fixed a few typos in the mdis2isis's default test. Fixes #4923.

### PR DESCRIPTION
**Pull request by kaitlyndlee**
fits2isis tests now check the labels. Also fixed a few typos in the mdis2isis's default test. Fixes #4923.

### CODE REVIEW

File: isis/src/base/apps/fits2isis/tsts/default/Makefile [hunk 1]
```diff
@@ -4,10 +4,6 @@ include $(ISISROOT)/make/isismake.tsts
 
 commands:
 	$(APPNAME) from=$(INPUT)/WFPC2u5780205r_c0fx.fits \
-	to=$(OUTPUT)/fitsTruth.cub \
-	> /dev/null;
-	raw2isis from=$(INPUT)/WFPC2u5780205r_c0fx.fits \
```
Comment (celbal) [inline isis/src/base/apps/fits2isis/tsts/default/Makefile:9]: Why did you remove this raw2isis call?

File: isis/src/messenger/apps/mdis2isis/tsts/default/Makefile [hunk 2]
```diff
@@ -10,9 +10,9 @@ commands:
 	  to=$(OUTPUT)/mdis2isisTruth.pvl> /dev/null
 	$(APPNAME) from=$(INPUT)/EN0089569526M.IMG \
 	  to=$(OUTPUT)/mdis2isisLutTruth.cub > /dev/null
-	catlab from=$(OUTPUT)/mdis2isisTruth.cub \
+	catlab from=$(OUTPUT)/mdis2isisLutTruth.cub \
```
Comment (celbal) [inline isis/src/messenger/apps/mdis2isis/tsts/default/Makefile:13]: Why Lut?

File: isis/src/messenger/apps/mdis2isis/tsts/default/Makefile [hunk 3]
```diff
@@ -10,9 +10,9 @@ commands:
...
 	  to=$(OUTPUT)/mdis2isisLutTruth.pvl> /dev/null
 	$(APPNAME) from=$(INPUT)/EN0131771763M.IMG \
 	  to=$(OUTPUT)/mdis2isisStrangeLabelTruth.cub > /dev/null
-	catlab from=$(OUTPUT)/mdis2isisTruth.cub \
+	catlab from=$(OUTPUT)/mdis2isisStrangeLabelTruth.cub \
```
Comment (celbal) [inline isis/src/messenger/apps/mdis2isis/tsts/default/Makefile:17]: Why StrangeLabel?

File: isis/src/base/apps/fits2isis/tsts/default/Makefile (hunk 1 above)
Comment (kaitlyndlee) [reply isis/src/base/apps/fits2isis/tsts/default/Makefile:9]: We don't do anything after the raw2isis call, so there is no need for it. The only reason I could see why we would call raw2isis is to check if the two output cubes are the same.

File: isis/src/messenger/apps/mdis2isis/tsts/default/Makefile (hunk 2 above)
Comment (kaitlyndlee) [reply isis/src/messenger/apps/mdis2isis/tsts/default/Makefile:13]: I changed it to Lut because if we used mdis2isisTruth.cub, we would be checking the label for mdis2isisTruth.cub again, since the first cat call does this, instead of the LutTruth cube we just created.

File: isis/src/messenger/apps/mdis2isis/tsts/default/Makefile (hunk 3 above)
Comment (kaitlyndlee) [reply isis/src/messenger/apps/mdis2isis/tsts/default/Makefile:17]: I changed it to StrangeLabel because if we used mdis2isisTruth.cub, we would be checking the label for mdis2isisTruth.cub again, since the first cat call does this, instead of the StrangeLabel cube we just created.

File: isis/src/messenger/apps/mdis2isis/tsts/default/Makefile (hunk 2 above)
Comment (celbal) [reply isis/src/messenger/apps/mdis2isis/tsts/default/Makefile:13]: What does Lut stand for though?

File: isis/src/messenger/apps/mdis2isis/tsts/default/Makefile (hunk 3 above)
Comment (celbal) [reply isis/src/messenger/apps/mdis2isis/tsts/default/Makefile:17]: Does the cube have a strange label then?

File: isis/src/messenger/apps/mdis2isis/tsts/default/Makefile (hunk 2 above)
Comment (kaitlyndlee) [reply isis/src/messenger/apps/mdis2isis/tsts/default/Makefile:13]: I am not sure. The pvl files were named this way before I made my changes. I renamed the cubes being passed into catlab to make sure it was testing the correct cube.

Comment (celbal) [reply isis/src/messenger/apps/mdis2isis/tsts/default/Makefile:13]: Is there a reasonable replacement for "Lut" that makes more sense in the context of this test? If not that is okay.

Comment (kaitlyndlee) [reply isis/src/messenger/apps/mdis2isis/tsts/default/Makefile:13]: Hi Makayla, nothing really stood out to me in the pvl file that would give me a new name.

**Review by celbal — APPROVED** [review]
